Src/ILGPU/ILGPU.csproj

```diff
@@ -145,11 +145,6 @@
           <AutoGen>True</AutoGen>
           <DependentUpon>IntrinsicMatchers.tt</DependentUpon>
         </Compile>
-        <Compile Update="Runtime\ExchangeBufferExtensions.cs">
-          <DesignTime>True</DesignTime>
-          <AutoGen>True</AutoGen>
-          <DependentUpon>ExchangeBufferExtensions.tt</DependentUpon>
-        </Compile>
```

Comment:
@Yey007 Looks like you removed too much. These lines should still be included.

In Visual Studio:
1. Right-click on `ExchangeBufferExtensions.tt` in the Solution Explorer.
1. Select `Run Custom Tool`
1. Go to the `File` menu, and select `Save All`.

The above lines should be added back to the project.

Reply:
Oh, I see. Right, I removed the wrong thing.